### PR TITLE
Add unit tests for new fields added for get account in PR#9838

### PR DESCRIPTION
**Pull request by nickjjzhao**
## Change Description
PR https://github.com/EOSIO/eos/pull/9838 populates few of the fields of get_account_results struct. Add unit tests to verify the populated values. 

## Testing Changes
**Select *ANY* that apply:**
- [x] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

### CODE REVIEW

File: plugins/chain_plugin/test/test_chain_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+    action_result stake( const account_name& from, const account_name& to, const asset& net, const asset& cpu ) {
+      return push_action( name(from), "delegatebw"_n, mvo()
+                          ("from",     from)
+                          ("receiver", to)
+                          ("stake_net_quantity", net)
+                          ("stake_cpu_quantity", cpu)
+                          ("transfer", 0 )
+      );
+    }
+    action_result stake( std::string_view from, std::string_view to, const asset& net, const asset& cpu ) {
```
Comment (venu-block1) [inline plugins/chain_plugin/test/test_chain_plugin.cpp:74]: Delete this unused overloaded stake() method.

File: plugins/chain_plugin/test/test_chain_plugin.cpp [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+    }
+
+    action_result stake( const account_name& acnt, const asset& net, const asset& cpu ) {
+      return stake( acnt, acnt, net, cpu );
+    }
+    action_result stake( std::string_view acnt, const asset& net, const asset& cpu ) {
+      return stake( account_name(acnt), net, cpu );
+    }
+
+    action_result vote( const account_name& voter, const std::vector<account_name>& producers, const account_name& proxy = name(0) ) {
```
Comment (venu-block1) [inline plugins/chain_plugin/test/test_chain_plugin.cpp:85]: Delete the unused stake() and vote() methods.

File: plugins/chain_plugin/test/test_chain_plugin.cpp (hunk 1 above)
Comment (nickjjzhao) [reply plugins/chain_plugin/test/test_chain_plugin.cpp]: Fixed.

File: plugins/chain_plugin/test/test_chain_plugin.cpp (hunk 2 above)
Comment (nickjjzhao) [reply plugins/chain_plugin/test/test_chain_plugin.cpp]: Fixed.

**Review by venu-block1 — APPROVED** [review]

File: plugins/chain_plugin/test/test_chain_plugin.cpp [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+    setup_system_accounts();
+    produce_blocks();
+    const asset nstake = core_from_string("1.0000");
+    const asset cstake = core_from_string("2.0000");
+    create_account_with_resources("alice1111111"_n, config::system_account_name);
+    //stake more than 15% of total EOS supply to activate chain
+    transfer( name("eosio"), name("alice1111111"), core_from_string("650000000.0000"), name("eosio") );
+
+    read_only::get_account_results results = get_account_info(name("alice1111111"));
+    BOOST_TEST_REQUIRE(results.total_resources.get_type() != fc::variant::type_id::null_type);
```
[the remark this answers is not in the document]
Comment (nickjjzhao) [reply plugins/chain_plugin/test/test_chain_plugin.cpp]: Fixed.

File: plugins/chain_plugin/test/test_chain_plugin.cpp [hunk 4]
```diff
@@ -0,0 +1,526 @@
...
+
+BOOST_AUTO_TEST_SUITE(chain_plugin_tests)
+
+BOOST_FIXTURE_TEST_CASE(account_results_total_resources_test, chain_plugin_tester) { try {
+
+    produce_blocks(10);
+    setup_system_accounts();
+    produce_blocks();
+    const asset nstake = core_from_string("1.0000");
+    const asset cstake = core_from_string("2.0000");
```
[the remark this answers is not in the document]
Comment (nickjjzhao) [reply plugins/chain_plugin/test/test_chain_plugin.cpp]: Fixed.

**Review by huangminghuang — APPROVED** [review]

**Pull request merged**